nanoribbon/viewers/show_computed.py

```diff
@@ -80,7 +81,7 @@ def __init__(self, **kwargs):
                                       layout=layout)
         kpoint_slider = ipw.IntSlider(description="k-point",
                                       min=1,
-                                      max=self.bands_array.shape[1],
+                                      max=nptk_ks , #self.bands_array.shape[1],
```

Comment:
```suggestion
                                      max=nptk_ks,
```

if the old thing was an error, you don't need to keep it as a comment.